pkg/connector/capabilities.go

```diff
@@ -30,7 +30,7 @@ func (*LinkedInConnector) GetCapabilities() *bridgev2.NetworkGeneralCapabilities
 }
 
 func (*LinkedInConnector) GetBridgeInfoVersion() (info, capabilities int) {
-	return 1, 2
+	return 1, 3
```

Comment:
`capID()` needs to be updated too (the version number bump tells the bridge to resend capabilities to all rooms, while capID is for detecting if the event in a specific room changed)